index.js

```diff
@@ -108,13 +109,17 @@ async function libraryHasChanged(name, libDir, targetDir, hashStore) {
 
 async function findFiles(libDir, targetDir) {
   const ignore = ["**/*", "!node_modules", "!.git"]
+  const { argv: { ignoreGit } } = yargs
```

Comment:
Could you register the option in cli.js instead of inline in this function? Otherwise the option would be shown in the help etc.